### PR TITLE
Updated the path of screenshots in readme

### LINKED ISSUE #962: Screenshots not visible in readme
**Actual Behaviour**

Screenshots are not visible due to wrong path

**Expected Behaviour**

Screenshots should be visible in readme

**Would you like to work on the issue?**

yes

### PR DESCRIPTION
**Pull request by vinaysajjanapu**
Fix #962 
Changes: 

Updated the wrong path of screenshots in readme.

Check preview here: https://github.com/vinaysajjanapu/phimpme-android/tree/ss_readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,11 @@ Join Gitter channel: https://gitter.im/fossasia/phimpme
...
-     <td><img src="/docs/screenshots/screenshot_3.png"></td>
-     <td><img src="/docs/screenshots/screenshot_5.png"></td>
-     </tr>
-     <tr>
-     <td><img src="/docs/screenshots/screenshot_2.png"></td>
-     <td><img src="/docs/screenshots/screenshot_4.png"></td>    
+     <td><img src="/docs/_static/screenshots/screenshot_1.png"></td>
+     <td><img src="/docs/_static/screenshots/screenshot_3.png"></td>
+     <td><img src="/docs/_static/screenshots/screenshot_5.png"></td>
+     <td><img src="/docs/_static/screenshots/screenshot_2.png"></td>
```
Comment (pa1pal) [inline README.md]: five screenshots in a row looks small. 
That's why I made two table of 3 and 2 screenshot. What do you think?

**Comment by mohitmanuja** [comment]
@pa1pal @vinaysajjanapu I think we should also add a screenshot of accounts.

**Comment by pa1pal** [comment]
@mohitmanuja Yes it is also a core feature, and two rows of three screenshots will look good.

**Comment by mohitmanuja** [comment]
@pa1pal Yes, Right. I will add a new screenshot with 2 rows in this PR.

**Comment by mohitmanuja** [comment]
@pa1pal I have made the changes.

**Pull request merged**